service/history/replication/task_processor.go

```diff
@@ -436,7 +436,7 @@ func (p *taskProcessorImpl) convertTaskToDLQTask(
 		}, nil
 
 	default:
-		return nil, fmt.Errorf("unknown replication task type: %v", replicationTask.TaskType)
+		return nil, fmt.Errorf("unknown replication task type: %T", replicationTask.TaskType)
```

Comment:
`%v` will also work as `TaskType` has `String()` method defined?

Reply:
Yes. Revert this one.